cherrypy/lib/httputil.py

```diff
@@ -66,6 +68,29 @@ def protocol_from_http(protocol_str):
     return int(protocol_str[5]), int(protocol_str[7])
 
 
+def passes_if_range_check(if_range_header):
+    """Determines if an If-Range header is present and passes its conditions"""
+    if if_range_header:
```

Comment:
Do
```suggestion
    if not if_range_header:
```
this will save us a lot of indentation.